packages/clerk-js/src/core/resources/SessionTask.ts

```diff
@@ -0,0 +1,56 @@
+import type {
+  ClerkOptions,
+  EnvironmentResource,
+  SessionTaskJSON,
+  SessionTaskKey,
+  SessionTaskResource,
+} from '@clerk/types';
+
+import { buildURL } from '../../utils';
+
+/**
+ * Maps URL paths for each session task key in order to not rely on FAPI nomenclature
+ */
+export const SESSION_TASK_PATH_BY_KEY: Record<SessionTaskKey, string> = {
+  org: 'add-organization',
+} as const;
+
+export class SessionTask implements SessionTaskResource {
+  key!: SessionTaskKey;
+
+  constructor(data: SessionTaskJSON) {
+    this.fromJSON(data);
+  }
+
+  protected fromJSON(data: SessionTaskJSON): this {
+    if (!data) {
+      return this;
+    }
+
+    this.key = data.key;
+
+    return this;
+  }
+
+  /**
+   * Resolves path for internal component routing
+   */
+  public __internal_getPath() {
```

Comment:
I'll move those to a separate module, either a singleton or simple `.ts`, cause it has nothing to do with the actual data resource but rather UI routing